Core/Core.xcodeproj/project.pbxproj

```diff
@@ -616,6 +616,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				MTL_FAST_MATH = YES;
+				ONLY_ACTIVE_ARCH = YES;
```

Comment:
Raises a few questions:

- Why did you need this setting?
- Does it override a default?
- Is this at target or project level? (sorry, I've not zoomed out or loaded the project locally myself to find out for lack of time)